src/test/resources/features/articles/createarticles.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Create articles
+
+  Background:
+    Given that I enter the conduit site
+    And I click on the Sign In link
+    And Authentic me on the site
+
+  @SuccessfulCreateArticle
```

Comment:
I think that some of these tags aren't quite useful. If a tag just runs an scenario (like this one), maybe we can think if it's useful or not.
I suggest to add some tag like @HappyPath.

Reply:
I understand, homologate the tags to indicate which is the happy path and which is the bad one